### ISSUE TITLE
Ensure extension added to "directory" imports

### ISSUE THREAD

**Issue by tpluscode** [karlprieb/babel-plugin-add-import-extension#1]
Given an import 

```ts
import { ScenarioStep } from '../'
```

It would be nice if the plugin transformed to 

```js
// or whatever extension configured
import { ScenarioStep } from '../index.js'
```

**Comment by karlprieb** [comment]
Ohh that's great :)
I'm not using this plugin on daily basis anymore, but I can add this in the next couple days or one week.
Thank you for your input!

**Comment by tpluscode** [comment]
Out of curiosity, what changed in your projects that you don't need this anymore?

**Comment by karlprieb** [comment]
I never used on work projects, because mainly we use some kind of bundler, but I used a lot for personal projects as all of them use esm. On my latest projects I used snowpack that already do that through esbuild I think. So maybe esbuild can do that (I'm not 100% sure).
But for projects that relies only on babel this project seems to be usefull as I didn't find anything on babel project that add the proper extensions and the typescript compiler will never do that as they already said on https://github.com/microsoft/TypeScript/issues/16577
:)
